frontends/infinite-corridor/package.json

```diff
@@ -19,6 +19,7 @@
     "@mui/material": "^5.9.0",
     "@swc/core": "^1.3.32",
     "@tanstack/react-query": "^4.29.5",
+    "@tanstack/react-query-devtools": "^4.29.6",
```

Comment:
Adds the [devtools](https://tanstack.com/query/v4/docs/react/devtools) for react query:

- only useable in our development build (`NODE_ENV=development`)
- does not significantly affect the production build. *It adds a few hundred **bytes**; [the devtools component](https://github.com/TanStack/query/blob/main/packages/react-query-devtools/src/index.ts) uses a `process.env.NODE_ENV !== 'development'` conditional that causes webpack to tree-shake it out in non-development builds. See screenshot of our bundle.*

**Dev tools:**

<img width="1728" alt="Screenshot 2023-05-16 at 7 27 51 PM" src="https://github.com/mitodl/open-discussions/assets/9010790/ca7c1d3e-1902-47ea-a403-a871054bae92">

<img width="1723" alt="Screenshot 2023-05-16 at 7 27 26 PM" src="https://github.com/mitodl/open-discussions/assets/9010790/0622cb88-8020-4a1d-a0f2-f52095e61d98">


**Affect on our bundle:**

<img width="936" alt="Screenshot 2023-05-16 at 7 18 36 PM" src="https://github.com/mitodl/open-discussions/assets/9010790/50ceb89c-3e82-4bc0-814f-3eb14368fc18">